### PR TITLE
Typescriptify the rest of the client!

### PR DESCRIPTION
**Pull request by hopskipnfall**
#61

### CODE REVIEW

**Comment by hopskipnfall** [comment]
Had to update to react-bootstrap v1 in this to make it work.

File: client/src/CreateBitModal.tsx [hunk 1]
```diff
@@ -26,28 +36,30 @@ class CreateBitModal extends Component {
           <h3>Create a Bit!</h3>
         </Modal.Header>
         <Modal.Body>
-          <LocalForm onSubmit={values => this.handleSubmit(values)} id="bit-form">
-            <ControlLabel> Title </ControlLabel>
+          <LocalForm onSubmit={values => this.handleSubmit(values)}
+          //  id="bit-form"
```
Comment (jimmyfnugent) [inline client/src/CreateBitModal.tsx:40]: Do we need this?

File: client/src/FilterControl.jsx [hunk 2]
```diff
@@ -10,7 +10,7 @@ const FilterControl = props => {
     filtering = Map(), toggleMainCharFilter, toggleVsCharFilter, toggleStageFilter, toggleStandaloneTagFilter,
   } = props;
   return (
-    <Panel bsStyle="primary" header={<h2> Filtering Options </h2>}>
+    <Panel variant="primary" header={<h2> Filtering Options </h2>}>
```
Comment (jimmyfnugent) [inline client/src/FilterControl.jsx:13]: Wait so do we need this and the TS version?

File: client/src/LoginButton.jsx [hunk 3]
```diff
@@ -6,7 +6,7 @@ import { initTwitterLogin } from './api_client';
 
 const LoginButton = props => {
   const {
-    profile, bsStyle = 'default', loginText = 'Log in with Twitter', fetchProfileIfNeeded,
+    profile, variant = 'default', loginText = 'Log in with Twitter', fetchProfileIfNeeded,
```
Comment (jimmyfnugent) [inline client/src/LoginButton.jsx:9]: Same question here

File: client/src/CreateBitModal.tsx (hunk 1 above)
Comment (hopskipnfall) [reply client/src/CreateBitModal.tsx]: Removed.

File: client/src/FilterControl.jsx (hunk 2 above)
Comment (hopskipnfall) [reply client/src/FilterControl.jsx]: Removed.

File: client/src/LoginButton.jsx (hunk 3 above)
Comment (hopskipnfall) [reply client/src/LoginButton.jsx:9]: Removed

**Pull request merged**